sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename

```diff
@@ -0,0 +1,132 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/start-spotenantrename
+applicable: SharePoint Online
+title: Start-SPOTenantRename
+schema: 2.0.0
+author: WayneEwington
+ms.author: waynewin
+ms.reviewer:
+---
+
+# Start-SPOTenantRename
+
+## SYNOPSIS
+
+ [!IMPORTANT]
```

Comment:
```suggestion
> [!IMPORTANT]
```

Anyway, this is the information I have about notes formatting in cmdlets articles:
"In cmdlet’s articles the [!NOTE] (also [!IMPORTANT] & [!CAUTION]) should not be used because it can’t be rendered later by the Get-Help cmdlet in the console window."